tests/Browser/Pages/Bookings.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Tests\Browser\Pages;
+
+use App\Models\Room;
+use Laravel\Dusk\Browser;
+
+class Bookings extends Page
+{
+    /**
+     * Get the URL for the page.
+     *
+     * @return string
+     */
+    public function url()
+    {
+        return '/bookings';
+    }
+
+    /**
+     * Assert that the browser is on the page.
+     *
+     * @param  Browser  $browser
+     * @return void
+     */
+    public function assert(Browser $browser)
+    {
+        $browser->assertPathIs($this->url());
+    }
+
+    /**
+     * Get the element shortcuts for the page.
+     *
+     * @return array
+     */
+    public function elements()
+    {
+        return [
+            '@element' => '#selector',
+        ];
+    }
+}
```

Comment:
This class can be removed, a `Page` is only useful if you are adding some re-used functionality (like filling a form for example, or clicking certain things)

Reply:
Removed the page. Once the tests pass feel free to approve.

Reply:
@alexstojda